documentation/api/io.strimzi.api.kafka.model.StrimziPodSet.adoc

```diff
@@ -0,0 +1,2 @@
+IMPORTANT: `StrimziPodSet` is an internal Strimzi resource.
+Strimzi users are not expected to create, modify or delete `StrimziPodSet` resources.
```

Comment:
Is it strong enough? What happens if they do?

Maybe something like:

IMPORTANT: `StrimziPodSet` is an internal Strimzi resource.
Information is provided for reference only.
Do not create, modify or delete `StrimziPodSet` resources as this might cause errors.